tools/scripts/upgrades/authz_upgrade_tx.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can you rename this file appropriately to start setting a standard others will follow?

For example: `authz_software_upgrade_tx_height_30_v_0_0_4.json`

Reply:
I will change the name to 'authz_upgrade_tx_example.json' because this is only an example.